std_msgs/Header header
uint8 status # 0 = UNKNOWN, 1 = AWAY, 2 = PRESENT